rhoas/api/api.go

```diff
@@ -5,12 +5,14 @@ import (
 	kafkainstanceclient "github.com/redhat-developer/app-services-sdk-go/kafkainstance/apiv1/client"
 	kafkamgmtclient "github.com/redhat-developer/app-services-sdk-go/kafkamgmt/apiv1/client"
 	svcacctmgmtclient "github.com/redhat-developer/app-services-sdk-go/serviceaccountmgmt/apiv1/client"
+	"github.com/redhat-developer/terraform-provider-rhoas/rhoas/localize"
 	"net/http"
 )
 
-type Clients interface {
+type Factory interface {
```

Comment:
Something out of scope of this PR that we should discuss later, since this interface is now called factory, we can probably refactor this and related packages a bit.